website/pages/docs/plugins/sources/_meta.json

```diff
@@ -15,6 +15,7 @@
   "hackernews": "Hacker News",
   "heroku": "Heroku",
   "k8s": "Kubernetes",
+  "launchdarkly": "LaunchDarkly",
```

Comment:
We need to add it to https://github.com/cloudquery/cloudquery/blob/06d8ff60dda19c6c7c325bd5e7cf10b3121bd524/website/pages/docs/plugins/sources/overview.mdx#L46 too

Reply:
Oops, done